Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -133,20 +134,23 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
                 parameter as string);
 
             IFileSystem fileSystem = FileSystem.Instance;
+            var sb = new StringBuilder();
+
+            var tool = Tool.CreateFromAssemblyData(typeof(AnalyzeCommand).Assembly, omitSemanticVersion: true);
```

Comment:
Was the new line addition here intentional?

Reply:
In fact, yes.. :) the whitespace here was intended to group certain things together. Whether that's improving readability or just making devs think the approach is inconsistent is an open question.